pkg/engine/harpoon.go

```diff
@@ -515,7 +467,25 @@ func (hc *HarpoonConfig) EngineMethod(ctx context.Context, path, method string,
 		return nil
 		// TODO
 	case kubeMethod:
-		if err := kubePodman(ctx, path); err != nil {
+		var path_ptr *string = nil
+		var prev *string = nil
+		if change != nil {
+			_, to, err := change.Files()
+			if err != nil {
+				log.Fatal(err)
+			}
+			if to != nil {
+				s, err := to.Contents()
+				if err != nil {
+					log.Fatal(err)
+				}
+				prev = &s
+			}
+		}
+		if path != "" {
```

Comment:
this will work, instead
```
if path == "" {
    return stopPods(ctx, []byte(*prev))
}
```
then, you won't need the `if path != nil` checks in kubePodman